lib/modules/platform/bitbucket-server/index.spec.ts

```diff
@@ -1002,6 +1055,138 @@ describe('modules/platform/bitbucket-server/index', () => {
         });
       });
 
+      describe('getUserSlugsByEmail', () => {
+        it('throws when lookup fails', async () => {
+          const scope = await initRepo();
+          scope
+            // User by email
+            .get(`${urlPath}/rest/api/1.0/users`)
+            .query(
+              (q) =>
+                q.filter === 'e-mail@test.com' &&
+                q['permission.1'] === 'REPO_READ' &&
+                q['permission.1.repositorySlug'] === 'repo' &&
+                q['permission.1.projectKey'] === 'SOME',
+            )
+            .reply(500, []);
+
+          await expect(
+            bitbucket.getUserSlugsByEmail('e-mail@test.com'),
+          ).rejects.toThrowErrorMatchingSnapshot();
```

Comment:
Better maintainable than snapshots:

```suggestion
          ).rejects.toThrow('Response code 500 (Internal Server Error)');
```